app/Http/Controllers/CourseController.php

```diff
@@ -59,33 +65,31 @@ public function create()
 	public function store(Request $request)
 	{
 		$attributes = request()->validate([
-			'courseName' => self::REQUIRED,
+			'courseName' => 'required',
 			'courseType' => 'required|integer|min:1|max:2',
 			'teamSize' => 'required|integer|min:1',
-			self::TEACHERS => 'required|array|min:1',
+			'teachers' => 'required|array|min:1',
 			'teachers.*' => [
 				'integer',
-				Rule::in(User::where('role', config(self::ROLES)[self::TEACHER])->get()->pluck('id')),
+				Rule::in(User::where('role', config(self::ROLES)['teacher'])->get()->pluck('id')),
```

Comment:
Qué hace esta validación? Igual y vale la pena poner un comentario porque no es inmediatamente obvio

Reply:
Valida que los ids que se estén leyendo del arreglo realmente correspondan a teachers y no a random users (en el caso de que alguien edite el select del front-end y agregue opciones). Si la consideras innecesaria la elimino, otherwise agrego el comentario dentro del archivo.

Reply:
El comment es suficiente :+1: